src/applications/ivc-champva/10-10D/config/form.js

```diff
@@ -150,7 +148,7 @@ const formConfig = {
         'I confirm that the identifying information in this form is accurate and has been represented correctly.',
       messageAriaDescribedby:
         'I confirm that the identifying information in this form is accurate and has been represented correctly.',
-      fullNamePath: formData => getNameKeyForSignature(formData),
+      fullNamePath: _formData => 'certifierName',
```

Comment:
Removed `getNameKeyForSignature()` since since we always want the `certifierName` as the form signature now.